web/src/scheduled_messages_ui.js

```diff
@@ -38,6 +38,7 @@ export function open_scheduled_message_in_compose(scheduled_msg, should_narrow_t
     if (scheduled_msg.type === "stream") {
         compose_args = {
             type: "stream",
+            stream_id: scheduled_msg.to,
             stream: sub_store.maybe_get_stream_name(scheduled_msg.to),
```

Comment:
Since we are yet to change `Filter` (which is used in `narrow.activate` to which these `compose_args` are passed) to use stream id instead of name, the stream name is indeed still required.

Reply:
This is true for both drafts and scheduled messages. Drafts already includes the stream id in the compose args so needs no tweaking for now.

Reply:
OK -- I think we probably do want to make the change I had, but then do a stream ID=>name conversion just before the `narrow.activate` calls, so that it's nicely localized (and easy to move when we fix `narrow.activate` to support taking stream IDs). We can do that in a follow-up PR I guess.